wiki/Gameplay/Full_combo/en.md

```diff
@@ -12,4 +12,4 @@ Scores that lost combo only via dropped slider ends are widely considered by the
 
 ![Screenshot of full combo and broken combo scores](img/combo-comparison.png "The top score is a full combo, and the bottom score is a broken combo.")
 
-Due to the [combo multiplier effect](/wiki/Gameplay/Combo_multiplier_effect), full combos will award the most [score](/wiki/Gameplay/Score) in [osu!](/wiki/Game_mode/osu!), [osu!taiko](/wiki/Game_mode/osu!taiko), and [osu!catch](/wiki/Game_mode/osu!catch) [game modes](/wiki/Game_mode).
+Because the [score](/wiki/Gameplay/Score) of an individual object in [osu!](/wiki/Game_mode/osu!) and [osu!catch](/wiki/Game_mode/osu!catch) [game modes](/wiki/Game_mode) heavily depends on combo multiplier and is virtually unbounded, full combos typically award the most score in these game modes.
```

Comment:
```suggestion
Because the [score](/wiki/Gameplay/Score) of an individual object in [osu!](/wiki/Game_mode/osu!) and [osu!catch](/wiki/Game_mode/osu!catch) heavily depends on combo multiplier and is virtually unbounded, full combos typically award the most score in these [game modes](/wiki/Game_mode).
```